src/docker_clojure/dockerfile/tools_deps.clj

```diff
@@ -36,7 +38,7 @@
          "RUN \\"]
         (concat-commands install-dep-cmds)
         (concat-commands
-          ["wget https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh"
+          ["curl -sLO https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh"
```

Comment:
To make my notes from https://github.com/docker-library/official-images/pull/16579#issuecomment-2048084513 more concrete, in case it's helpful: :heart:

```suggestion
          ["curl -fsSLO https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh"
```